### ISSUE TITLE
Feature suggestions for accessibility for people with low vision/blindness

### ISSUE THREAD

**Issue by Alwaysfindtheway** [kwaroran/RisuAI#170]
AI chats are built by a diverse group of people from around the world. RisuAI's community is built by people all over the world.

AI chat is a community and a hobby that is accessible to people with low vision and blindness using screen readers, especially since it is almost entirely written.

Therefore, I would like to make the following suggestions for improving RisuAI's accessibility features.

[ Alternative text is not provided for several features in RisuAI ]

1. Character bot cards are not provided with alt text, making them impossible to distinguish for users using screen readers.
- Solution: Modify the code to provide alt text in the portion of the bot card's EXIF that is provided by Character Name={{char}}.

2. All toggle-type checkbox features within RisuAI are currently unable to recognize whether they are checked or unchecked by screen readers.
 - Solution: I'm not sure if this is a technical issue or not, but it appears that screen readers are not currently recognizing these checkboxes.
 
For more details and English descriptions, see the links below.
 - https://arca.live/b/characterai/77819785
 - https://arca.live/b/characterai/78116226
 - https://arca.live/b/characterai/78392403

Please help the community of RisuAI get more people, and more diverse people, involved.
Thank you for your service as always.
